src/aap_eda/settings/default.py

```diff
@@ -319,7 +319,11 @@ def _get_secret_key() -> str:
 RQ_PERIODIC_JOBS = [
     {
         "func": "aap_eda.tasks.orchestrator.monitor_rulebook_processes",
-        "interval": 5,
+        "interval": 600,
```

Comment:
@Alex-Izquierdo What we need to know is when the activation process ends. For that the event monitor will trigger a refresh as soon as the container ends which is better than what we have today we have to wait for the monitor to wake up and detect that the container has ended..The even monitor relegates the periodic monitor to run with longer delays since it doesn't play an important role anymore.

Reply:
Should I guess there is a missing periodic job for `monitor_engine_events` ?
Even in that case we still have the unresponsive case and regular logs updates. You want to update the logs each 10 minutes or force it to be a manual sync? that's a bad UX IMO.

Reply:
@Alex-Izquierdo The design allows for the UI to mimic the periodic 10 second monitoring that it can call on the activation that will fetch the logs. There is a new end point /activations/<nn>/monitor which the UI can call when a user is involved. When there is no user involved and we are running with lights out or the log level set to Error there is no need to be wasting CPU cycles.